src/save_index.cpp

```diff
@@ -46,6 +46,16 @@ void replace_space2underbar(std::string &name) {
 	std::replace(name.begin(),name.end(),' ','_');
 }
 
+/** Replace the spaces in the filter */
+std::string replace_filter(const std::string* filter)
```

Comment:
Also, it's not a huge deal, but it would be better to pass by reference instead of pointer (`&` instead of `*`).